legal-api/src/legal_api/resources/business/business_filings.py

```diff
@@ -501,9 +501,12 @@ def _create_invoice(business: Business,  # pylint: disable=too-many-locals
         if filing.filing_type == Filing.FILINGS['incorporationApplication'].get('name'):
             mailing_address = Address.create_address(
                 filing.json['filing']['incorporationApplication']['offices']['registeredOffice']['mailingAddress'])
-            corp_type = filing.json['filing']['incorporationApplication']['nameRequest']['legalType']
-            business.legal_name = filing.json['filing']['incorporationApplication']['nameRequest'].get('legalName') \
-                or business.identifier
+            corp_type = filing.json['filing']['business'].get('legalType', 'BC')
```

Comment:
If the filing.json is an incorp filing then you won't have a business section according to the schema, correct? This should fail the pytests if they're using the example data from the schema repo

Reply:
maybe we should update the schema...hmmm...but the sample data has been changed and we can be guaranteed of a 'business' node since the introduction of the bootstrap